### DISCUSSION TITLE
Adding custom blocks and ability to add them to pages

### DISCUSSION THREAD

**Discussion by fyrepenguin** [factly/dega#473] [Ideas]
We can give users the ability to build their own blocks like subscription forms, custom banners, custom scripts or html that can be used in multiple posts or pages.

**Comment by deshetti** [comment]
This might be worth checking to see if we could implement it according to their spec: 

Intro: https://blockprotocol.org/
Specification: https://blockprotocol.org/spec
Developing Blocks: https://blockprotocol.org/docs/developing-blocks#the-quickest-start
